### DISCUSSION TITLE
Idea: The ability to choose between a standard map and an extended map in DOTA 2.

### DISCUSSION THREAD

**Discussion by 7ReaperOfSouls7** [ValveSoftware/Dota2-Gameplay#9814] [Ideas]
Hello Dear Support!
I have an idea for the development of DOTA 2.

I propose to return the standard (old) card to the game along with the past (namely: random drop, no tokens, display of all common items, display in the store which players have which items) the functionality of neutral items.
Make two modes to choose from: Standard Map(old generic neutral item store functionality) and Extended Map(new generic neutral item store functionality)
Based on many feedback (on Steam) from players after the release of New Horizons.
Many players do not like the extended map.
A large number of players left the game in protest.
There was a lot of negativity about the extended map.
In this regard, players should have a choice between the standard map and the extended map.
Those who like the extended map with tokens will play on it.
And whoever likes the standard map with random neutral items the most will play on this map.
There is a high probability that players who left the game due to the expanded map will return to the game again.
Online, respectively, will grow and developers and players will be in the black.

Please forward my idea to the developers for consideration.
I hope for your understanding.
All the best!

**Comment by 7ReaperOfSouls7** [comment]
Additional Information:
What will the addition of a new mode in the form of a standard map give (the old functionality of the storage of neutral items):

1. Online will increase. The players who left the game will return.
2. New mode.
3. Everyone will be in the black.
4. There will be less negativity from players as there will be map selection.
For example, in Heroes Of The Storm there are more than 10 maps for battles, and in DOTA 2 there is still one map for all game modes.
5. An example from life. If many people smoke, drink and swear, this does not mean that all people should smoke, drink and swear.
There are also people who have made the choice not to smoke, not to drink and not to swear.
It's the same in DOTA 2. Each player should have a choice in which mode to play and on which map to play.

P.S. I think that my idea is quite normal and interesting, which can be added to the game for more variety in the choice of modes and maps.

**Comment by Yulaise** [comment]
afterwards I like to have some fun by playing some of the games that are available in the [free games](https://freegamesonline.io/)  section of the website.
